addons/udes_stock/models/stock_move.py

```diff
@@ -195,16 +195,17 @@ def split_out_move_lines(self, move_lines):
                 }
             )
 
+            if self.move_orig_ids:
+                (bk_move | self).update_orig_ids(self.move_orig_ids)
+
             # When not complete, splitting a move may change its state,
             # so recompute
             incomplete_moves = (self | bk_move).filtered(lambda mv: mv.state != "done")
+            incomplete_moves.recompute()
             incomplete_moves._recompute_state()
 
             move_lines.write({"state": bk_move.state})
```

Comment:
Does this also need moving?

Reply:
I don't think so, we first recompute the state of the stock.move and then update its move lines.

Reply:
Is the state of the moves dependent on the move lines?

Reply:
Yes, but not on their state.